DataFormats/EcalDetId/interface/EcalTrigTowerDetId.h

```diff
@@ -2,41 +2,87 @@
 #define ECALDETID_ECALTRIGTOWERDETID_H
 
 #include <iosfwd>
+#include <cassert>
 #include "DataFormats/DetId/interface/DetId.h"
 #include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
-
+#include "FWCore/Utilities/interface/Exception.h"
 
 /** \class EcalTrigTowerDetId
     
    DetId for an Ecal Trigger tower
 
 */
 
+// a workaround for std::abs not being a constexpr function
```

Comment:
std::abs is not constexpr...  this is a workaround
i do not know if this gonna have a penalty or not